README.md

```diff
@@ -56,6 +56,22 @@ doc = DocumentFile.from_pdf("path/to/your/doc.pdf").as_images()
 result = model(doc)
 ```
 
+### Deal with the rotation issues
+You can provide rotated documents, and/or documents with different boxes orientations.
+We provide different options to deal with such cases:
+
+- If you will only use straight document pages with straight words (horizontal, same reading direction),
+pass `assume_straigth_pages=True` in the `ocr_predictor`, it will directly fit straight boxes to your pages
+and return straight boxes, it is the faster pipeline.
```

Comment:
"consider passing `assume_straight_boxes=True` to the `ocr_predictor`. It will directly fit straight boxes on your page and return straight boxes, which makes it the fastest option."